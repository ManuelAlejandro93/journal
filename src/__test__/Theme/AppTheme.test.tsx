import { render } from '@testing-library/react';
import { AppTheme } from '@/Theme';

describe('Prueba sobre <AppTheme>', () => {
  test('Debe ser renderizable', () => {
    const { container } = render(
      <AppTheme>
        <></>
      </AppTheme>
    );
    expect(container).toBeTruthy();
  });
  test('Debe poder tener un snapshot', () => {
    const { container } = render(
      <AppTheme>
        <h1>Soy un titulo generico.</h1>
      </AppTheme>
    );

    expect(container).toMatchSnapshot();
  });
});
