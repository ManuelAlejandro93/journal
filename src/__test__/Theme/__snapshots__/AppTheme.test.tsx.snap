// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Prueba sobre <AppTheme> Debe poder tener un snapshot 1`] = `
<div>
  <h1>
    Soy un titulo generico.
  </h1>
</div>
`;
